projects/packages/waf/src/class-rest-controller.php

```diff
@@ -170,8 +170,8 @@ public static function update_waf( $request ) {
 			}
 		}
 
-		// Only attempt to update the WAF if the module is supported
-		if ( Waf_Runner::is_supported_environment() ) {
+		// Only attempt to update the WAF if the module is supported and enabled.
+		if ( Waf_Runner::is_supported_environment() && Waf_Runner::is_enabled() ) {
```

Comment:
I'm wondering if https://github.com/Automattic/jetpack/pull/38326 makes this change unnecessary, I will double check.